webpack/app/js/contrast.js

```diff
@@ -1,31 +1,80 @@
 import 'jquery.cookie';
 
-
+// Adciona alto contras a página e inverte os ícones para sua versão de alto contraste
 jQuery(function($) {
   $('.link-contraste').click(
     function(e) {
+      console.log($.cookie('contraste'));
       if($.cookie('contraste') === null) {
         $.cookie('contraste', 'on');
-        $('body').addClass('contraste');
+
+        $('body').addClass('contraste').addClass('inverter');
```

Comment:
Por que precisamos da class ``inverter``? Somente a class ``contraste`` não seria o suficiente?

Voce chegou a ver este arquivo com regras especificas para contraste? https://github.com/plonegovbr/brasil.gov.temas/blob/master/webpack/app/scss/_contrast.scss

Reply:
usei a classe inverter justamente pra ter um controle maior sobre o que eu estava fazendo e não "bagunçar" o que estava feito na classe "contraste". Mas posso converter tudo que for "inverter" para a classe "contraste" e colocar neste arquivo, sem grandes problemas.

Essa é uma branch em desenvolvimento ainda, e faltam alguns testes a serem realizados..

Reply:
Ainda não consigo entender por que precisa de todas essas classes, para mim codigo como estava ja era o suficiente, essa atividade não deve envolver javascript.

Vamos manter as coisas simples por favor.

Reply:
alguns elementos precisam de um tratamento melhor do que só mudar a  cor do fundo, existem vários elementos que ao se aplicar o alto contraste ficam ilegíveis ou ferem as regras de contraste-alto contraste, esse branch é um trabalho em desenvolvimento, não é versão final e ainda será(bastante) ajustado, não encare essas alterações como finais, são o que tem que ser, testes de estilos sobre elementos do sistema em duas situações diferentes.

Somente comitei e fiz o push dessas alterações para salvar o progresso até aqui e não perder o que foi feito, tendo em vista que minha estação não é muito confiável e esta não é a única demanda que estou atendendo.